lib/Widget/material.dart

```diff
@@ -107,4 +108,134 @@ class CustomMaterial {
+    if (now.isAfter(formTime1) && check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableVirtalSignButton2(bool check) {
+    DateTime now = DateTime.now();
+    DateTime formTime2 = DateFormat.Hm().parse("06:00");
+    formTime2 = new DateTime(
+        now.year, now.month, now.day, formTime2.hour, formTime2.minute);
+    if (now.isAfter(formTime2) && check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableVirtalSignButton3(bool check) {
+    DateTime now = DateTime.now();
+    DateTime formTime3 = DateFormat.Hm().parse("10:00");
+    formTime3 = new DateTime(
+        now.year, now.month, now.day, formTime3.hour, formTime3.minute);
+    if (now.isAfter(formTime3) && check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableVirtalSignButton4(bool check) {
+    DateTime now = DateTime.now();
+    DateTime formTime4 = DateFormat.Hm().parse("14:00");
+    formTime4 = new DateTime(
+        now.year, now.month, now.day, formTime4.hour, formTime4.minute);
+    if (now.isAfter(formTime4) && check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableVirtalSignButton5(bool check) {
+    DateTime now = DateTime.now();
+    DateTime formTime5 = DateFormat.Hm().parse("18:00");
+    formTime5 = new DateTime(
+        now.year, now.month, now.day, formTime5.hour, formTime5.minute);
+    if (now.isAfter(formTime5) && check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableVirtalSignButton6(bool check) {
+    DateTime now = DateTime.now();
+    DateTime formTime6 = DateFormat.Hm().parse("22:00");
+    formTime6 = new DateTime(
+        now.year, now.month, now.day, formTime6.hour, formTime6.minute);
+    if (now.isAfter(formTime6) && check == false) {
+      return true;
+    }
+
+    return false;
+  }
+
+  bool disableRespiratoryDay0Button(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableUrologyButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableBloodClotButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableDrainButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableNutritionButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableRespiratoryDay1Button(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableDigestiveButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disableInfectionButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
+
+  bool disablePulmanaryButton(bool check) {
+    if (check == false) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
Make this as `[sth]_util.dart` Since it only manage 1 or 2 file(s). Material.dart should be something about color / EdgeInsets bla bla.

Reply:
Or it can be viewmodel idk

Reply:
I move that part of code as a view model file.